src/info.json

```diff
@@ -17,6 +17,7 @@
         "(?)MomoTweak",
         "(?)angelspetrochem",
         "(?)SuperExpensiveMode",
-        "(?)H2O"
+        "(?)H2O",
+	"(?)deadlock_stacked_recipes"
```

Comment:
You used a tab instead of spaces. Either update the pull request or I will do it. After that I will accept the PR.

Reply:
grrr, silly vim